uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/leave-request-instance_test.js

```diff
@@ -128,19 +115,6 @@ define([
             });
           });
         });
-
-        describe('error', function () {
-
-          beforeEach(function () {
-            commonSetup('approved', 'approve', leaveRequestMockData.singleDataError());
-          });
-
-          it('updates the status_id of the instance', function () {
```

Comment:
Were this `it` titles wrong? It doesn't make sense that with "error", "it updates the status_id of the instance"